server/src/main/resources/application.properties

```diff
@@ -16,6 +16,7 @@ springdoc.api-docs.path=/api-docs
 loadSample=true
 
 conductor.db.type=memory
+conductor.queue.type=redis_standalone
```

Comment:
Does this add redis standalone dependency to starting Conductor?

Reply:
yes @aravindanr , this will add the dependency on redis standalone

Reply:
@manan164 is there a chance you could update this [doc](https://conductor.netflix.com/devguide/running/source.html) to mention we need to run redis standalone before we start server locally?

Reply:
if i didn't define this config `conductor.queue.type`, will I still use the orkes_queue?

Reply:
No @lijia-rengage , We need to mention the queue type otherwise it will use dynomite.